requirements-integration.in

```diff
@@ -1,6 +1,6 @@
 aiohttp
 jinja2
-juju<3.1
+juju<3.0
```

Comment:
Could you please add a comment here saying we need this version of juju because 3.0 is not maintained anymore?

Reply:
Done.